lib/capitano/resin-toolbox.coffee

```diff
@@ -201,7 +201,16 @@ module.exports =
 		{ runtimeOptions, configYml } = parseOptions(options, params)
 
 		if not fileExists(path.join(runtimeOptions.baseDir, 'Dockerfile'))
-			throw new Error("No Dockerfile was found in the project directory: #{runtimeOptions.baseDir}")
+			if fileExists(path.join(runtimeOptions.baseDir, 'Dockerfile.template'))
+				throw new Error('Dockerfile.template files are not yet supported by local push.
+
+					\n\nAs a workaround, you can rename your template to just \'Dockerfile\', and replace %TEMPLATE%
+					strings with a fixed value. For example \'%RESIN_MACHINE_NAME%\' might become
```

Comment:
Shouldn't these use double `%`?
Like `%%RESIN_MACHINE_NAME%%` in:
* https://github.com/resin-io-projects/simple-server-node/blob/master/Dockerfile.template#L5
* https://docs.resin.io/deployment/docker-templates/

Reply:
Also how about something move invoking like:
> As a workaround, you can rename your \'Dockerfile.template\' to \'Dockerfile\', and replace all %%TEMPLATE%% strings with the appropriate values, as documented in https://docs.resin.io/deployment/docker-templates/ .

Reply:
Good points! This review should definitely 'request changes' though - that first point is absolutely a mistake :smiley:.

I've now fixed the %% part, and for the 2nd comment, I've taken most of your text, but left in the `raspberrypi3` example as well - I'd like that have the directly included in the output because I expect 90% of people hitting this need exactly that one change, and I'd like to make it as easy as possible for them. Sound ok?